src/theme/tokens/space.ts

```diff
@@ -14,6 +14,7 @@ export const space = {
   8: "2rem",
   9: "2.25rem",
   10: "2.5rem",
+  11: "2.75rem",
```

Comment:
You should be able to remove this once you've addressed height via my comment in the button theme file.